### PR TITLE
QA reviews

### PR DESCRIPTION
**Pull request by g4s8**
zerocracy/farm#646 - QA reviews schema, tests and xsl

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#302](https://github.com/zerocracy/datum/issues/302) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#302](https://github.com/zerocracy/datum/issues/302) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/570657051). The budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

**Comment by carlosmiranda** [comment]
@yegor256 looks good to merge

File: xsd/pm/qa/reviews.xsd [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  <xs:include schemaLocation="http://datum.zerocracy.com/SNAPSHOT/xsd/basics.xsd"/>
+  <xs:include schemaLocation="http://datum.zerocracy.com/SNAPSHOT/xsd/pm/scope/types.xsd"/>
+  <xs:simpleType name="quality">
+    <xs:restriction base="xs:string">
+      <xs:pattern value="(good|bad|acceptable)"/>
+    </xs:restriction>
+  </xs:simpleType>
+  <xs:complexType name="review">
+    <xs:sequence>
+      <xs:element name="quality" type="quality" minOccurs="0"/>
```
Comment (yegor256) [inline xsd/pm/qa/reviews.xsd]: @g4s8 I think we will also need:

 * `requested`: date/time when we asked the inspector to do the review
 * `inspector`: user name of the inspector
 * `done`: date/time when it was completed

Make sense?

**Comment by yegor256** [comment]
@g4s8 see above, I'm sure we will need that info

**Comment by g4s8** [comment]
@yegor256 fixed, please see new changes

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Order was finished, quality was "acceptable": +15 points just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
The job [#302](https://github.com/zerocracy/datum/issues/302) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by yegor256** [comment]
@rultor release, tag is `0.57`

**Pull request merged**